submit_and_compare/public/view.less

```diff
@@ -0,0 +1,67 @@
+.submit_and_compare {
+    .question_prompt {
+        text-align: left;
+    }
+    .student_answer {
+        color: #3399cc;
+        font-weight: 600;
+        padding-bottom: 20px;
+        padding-top: 20px;
```

Comment:
```suggestion
        padding-top: 20px;
        padding-bottom: 20px;
        color: #3399cc;
        font-weight: 600;
```

Reply:
I'm guessing that your comments throughout this file are a request to group properties by type?

Side note: The new Github PR "Suggested Change" feature is nice, but comments be still often be useful;
it's not obvious what you're asking for here.

Anyway, assuming it's the grouping, that's not prescribed by our style guide nor edx's.

It's also not enforced by any linter of which I'm aware.
I think you made a similar comment on another XBlock PR, but our linter actually _does_ alphabetize properties, without concern to type grouping.

While there some general support for property grouping, I don't believe it's a standard (like pylint/pycodestyle) and since none of our groups enforce it and neither does our linter, I don't think we should make these changes.